src/django/api/templates/mail/claim_facility_denial_body.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>
+            Your request to claim an Open Apparel Registry facility was denied
+        </title>
+    </head>
+    <body>
+        <p>
+            Hi,
+        </p>
+        <p>
+            You're receiving this email because we denied your request to claim this
+            Open Apparel Registry facility:
+        </p>
+        <ul>
+            <li>
+                Facility: {{ facility_name }}, {{ facility_address }}
+            </li>
+            <li>
+                Facility URL: {{ facility_url }}
+            </li>
+        </ul>
+        {% if len(denial_reason) %}
```

Comment:
I am running into a `Could not parse the remainder: '(denial_reason)' from 'len(denial_reason)'` exception when the deny logic. I don't think we have access to `len` in a template. We likely have to use the `|length` filter instead

```diff
diff --git a/src/django/api/templates/mail/claim_facility_denial_body.html b/src/django/api/templates/mail/claim_facility_denial_body.html
index 6bef756..04da155 100644
--- a/src/django/api/templates/mail/claim_facility_denial_body.html
+++ b/src/django/api/templates/mail/claim_facility_denial_body.html
@@ -21,7 +21,7 @@
                 Facility URL: {{ facility_url }}
             </li>
         </ul>
-        {% if len(denial_reason) %}
+        {% if denial_reason|length %}
         <p>
             Here's the reason for the denial:
         </p>
diff --git a/src/django/api/templates/mail/claim_facility_denial_body.txt b/src/django/api/templates/mail/claim_facility_denial_body.txt
index ea32f30..ad13953 100644
--- a/src/django/api/templates/mail/claim_facility_denial_body.txt
+++ b/src/django/api/templates/mail/claim_facility_denial_body.txt
@@ -6,7 +6,7 @@ You're receiving this email because we denied your request to claim this Open Ap
 - Facility: {{ facility_name }}, {{ facility_address }}
 - Facility URL: {{ facility_url }}
 
-{% if len(denial_reason) %}
+{% if denial_reason|length %}
 Here's the reason for the denial:
 
 {{ denial_reason }}

```

```
ERROR:django.request:Internal Server Error: /api/facility-claims/1/deny/
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/django/template/smartif.py", line 175, in translate_token
    op = OPERATORS[token]
KeyError: 'len(denial_reason)'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/rest_framework/viewsets.py", line 116, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/usr/local/lib/python3.7/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python3.7/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.7/contextlib.py", line 74, in inner
    return func(*args, **kwds)
  File "/usr/local/src/api/views.py", line 1409, in deny_claim
    send_claim_facility_denial_email(request, claim)
  File "/usr/local/src/api/mail.py", line 72, in send_claim_facility_denial_email
    text_template = get_template('mail/claim_facility_denial_body.txt')
  File "/usr/local/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/usr/local/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/usr/local/lib/python3.7/site-packages/django/template/engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "/usr/local/lib/python3.7/site-packages/django/template/engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "/usr/local/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 198, in compile_nodelist
    return parser.parse()
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python3.7/site-packages/django/template/loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python3.7/site-packages/django/template/defaulttags.py", line 955, in do_if
    condition = TemplateIfParser(parser, bits).parse()
  File "/usr/local/lib/python3.7/site-packages/django/template/defaulttags.py", line 888, in __init__
    super().__init__(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/template/smartif.py", line 166, in __init__
    mapped_tokens.append(self.translate_token(token))
  File "/usr/local/lib/python3.7/site-packages/django/template/smartif.py", line 177, in translate_token
    return self.create_var(token)
  File "/usr/local/lib/python3.7/site-packages/django/template/defaulttags.py", line 891, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 568, in compile_filter
    return FilterExpression(token, self)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 668, in __init__
    "from '%s'" % (token[upto:], token))
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '(denial_reason)' from 'len(denial_reason)'
```

Reply:
Fixed in b289c2626f51de30ae1a4fb1d766c94e4434e310